app/web/features/search/SearchResultsList.tsx

```diff
@@ -118,19 +119,21 @@ export default function SearchResultsList({
   const hasAtLeastOnePageResults =
     results && results?.pages[0]?.resultsList?.length !== 0;
 
-  let resultsList: any = results?.pages
+  let resultsList = results?.pages
     .flatMap((page) => page.resultsList)
     .filter((result) => result.user);
 
   let wasResultFound = false;
-  resultsList?.map((value: any) => {
+
+  // @TODO - Can make this one variable by using a find function, resultsList.find, etc.
+  resultsList?.map((value) => {
     if (value.user?.userId === selectedResult?.userId) {
       wasResultFound = true;
     }
   });
```

Comment:
I believe you can turn this into one variable like:

```javascript
const wasResultFound = resultsList?.find((value) => value.user?.userId === selectedResult?.userId) !== undefined;
```

It's a bit more performant as find will stop as soon as the first value is true:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
I guess that instead of `!== undefined` should be `false`?? (if I'm understanding properly)

Reply:
.find will return undefined if nothing passes the given test. Since you want it to return a boolean I added the !== undefined. So if it found something then it’s not equal to undefined it will return true so wasResultFound = true.

Does that make sense? Not sure I explained well.

Reply:
sure, absolutely clear, will update it now!

Reply:
done ✅